queries/indents.scm

```diff
@@ -6,7 +6,7 @@
   (parameter_call_list)
   (function_call)
   (class_def)
-  (class_var)
+  (classvar)
```

Comment:
Fixes
```
Query error at indents.scm:9. Invalid node type class_var
```

Reply:
Nice Find! I'm curious - how are you testing this since you are getting those errors?